### PR TITLE
Remove SFP sensors from sensors.conf

### PR DESCRIPTION
**Pull request by keboliu**
##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn2010-r0/sensors.conf [hunk 1]
```diff
@@ -47,3 +47,25 @@ bus "i2c-2" "i2c-1-mux (chan_id 1)"
         label fan2 "Chassis Fan 2"
         label fan3 "Chassis Fan 3"
         label fan4 "Chassis Fan 4"
+        ignore temp2
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-mlnx_msn2010-r0/sensors.conf:50]: I remember there are 16 ports in 2010, but there are 20 ignore here

File: device/mellanox/x86_64-mlnx_msn2700a1-r0/sensors.conf [hunk 2]
```diff
@@ -21,6 +21,42 @@ bus "i2c-17" "i2c-1-mux (chan_id 8)"
 # Temperature sensors
 chip "mlxsw-i2c-*-48"
     label temp1 "Ambient ASIC Temp"
+    ignore temp2
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-mlnx_msn2700a1-r0/sensors.conf:24]: There are 32 ports in 2700a1, but there are 35 ignore here

File: device/mellanox/x86_64-mlnx_msn2740-r0/sensors.conf [hunk 3]
```diff
@@ -74,3 +74,39 @@ bus "i2c-2" "i2c-1-mux (chan_id 1)"
         label fan2 "Chassis Fan Drawer-2"
         label fan3 "Chassis Fan Drawer-3"
         label fan4 "Chassis Fan Drawer-4"
+        ignore temp2
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-mlnx_msn2740-r0/sensors.conf:77]: There are 32 ports in 2740, but there are 35 ignore here

File: device/mellanox/x86_64-mlnx_msn3800-r0/sensors.conf [hunk 4]
```diff
@@ -8,6 +8,38 @@
 bus "i2c-2" "i2c-1-mux (chan_id 1)"
     chip "mlxsw-i2c-*-48"
         label temp1 "Ambient ASIC Temp"
+        ignore temp2
```
Comment (Junchao-Mellanox) [inline device/mellanox/x86_64-mlnx_msn3800-r0/sensors.conf:11]: There are 64 ports in 2740, but there are 32 ignore here

File: device/mellanox/x86_64-mlnx_msn2010-r0/sensors.conf (hunk 1 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn2010-r0/sensors.conf:50]: 2010 has 22 ports, 18 25G + 4 100G, so we need to ignore 22 sensors here

File: device/mellanox/x86_64-mlnx_msn2700a1-r0/sensors.conf (hunk 2 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn2700a1-r0/sensors.conf:24]: changed

File: device/mellanox/x86_64-mlnx_msn2740-r0/sensors.conf (hunk 3 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn2740-r0/sensors.conf:77]: changed

File: device/mellanox/x86_64-mlnx_msn3800-r0/sensors.conf (hunk 4 above)
Comment (keboliu) [reply device/mellanox/x86_64-mlnx_msn3800-r0/sensors.conf:11]: fixed

**Review by Junchao-Mellanox — APPROVED** [review]

**Pull request closed without merging**